torch/csrc/jit/codegen/cuda/lower_loops.cpp

```diff
@@ -303,15 +304,234 @@ void LoopNestGenerator::handle(Expr* expr) {
     popFor();
 }
 
+namespace {
+
+TensorView* findOutputTensor(Expr* expr) {
```

Comment:
do we want to assert here on `outputs().size() > 1`? I don't believe we have any today, but would provide a better error if we add one at some point.

Reply:
Added. Thanks.